getting-started/getting-started/SceneDelegate.swift

```diff
@@ -46,12 +43,16 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
       )
       
       // Preloads dummy data
-      preloadData(provider)
-      
-      // Assigns to SceneDelegate for future use
-      chatProvider = provider
+      preloadData(provider) { [weak self] in
+        self?.chatProvider = provider
+        self?.didPreloadData(windowScene: windowScene)
+      }
+    } else {
+      didPreloadData(windowScene: windowScene)
     }
-    
+  }
+  
+  func didPreloadData(windowScene: UIWindowScene) {
```

Comment:
Smallest of requests, but can this method be renamed to better specify its purpose in displaying/building the UI?

Reply:
sure thing, is `setupView(with windowScene: UIWindowScene)` ok for you?